src/dlgTriggerEditor.cpp

```diff
@@ -70,11 +70,14 @@ const QString msgInfoAddTimer = "Check the manual for <a href='http://wiki.mudle
 const QString msgInfoAddButton = "To add a new button: <b>1.</b> Add a new group to define a new Button bar in case you don't have any."
                                  "<b>2.</b> Add new buttons to a button bar."
                                  "Check the manual for <a href='http://wiki.mudlet.org/w/Manual:Contents'>more information</a>.";
-
-const QString msgInfoAddKey = "To add a new key binding <b>1.</b> add a new key <b>2.</b> click on <u><b>grab key</b></u> and then press your key combination. <b><u>NOTE:</u></b> If you want to bind "
-                              "a key combination you must hold down the modifier keys (e.g. control, shift etc.) down before clicking on grab key. "
-                              "<b>3.</b> Define a command that is executed when the key is hit. <b>4. <u>Activate</u></b> the new key binding."
-                              "Check the manual for <a href='http://wiki.mudlet.org/w/Manual:Contents'>more information</a>.";
+const QString msgInfoAddKey = "<p>To add a new key binding:"
+                              "<ol><li><b>1.</b> add a new key</li>"
+                              "<li><b>2.</b> click on <b><tt>Click to grab key</tt></b> and then press your key combination. "
+                              "<i>NOTE: If you want to bind a key combination you must hold down the modifier keys (e.g. <tt>Control</tt>, <tt>Shift</tt> etc.) down before clicking on grab key.</i></li>"
+                              "<li><b>3.</b> define a command that is executed when the key is hit.</li>"
+                              "<li><b>4. <u>Activate</u></b> the new key binding.</li>"
+                              "<li><b>5.</b> if you wish to adjust the modifiers subsequently you may click on the check boxes but be aware that some modifiers may not be available on all operating systems or may be used for some keys on an operating system outside of a user's control.</ol></p>"
```

Comment:
> be aware that some modifiers may not be available on all operating systems or may be used for some keys on an operating system outside of a user's control.

it's not feedback that a user can actually action on. sure, it's a caution, that's nice - but as a user, I'll just go "ok? what am I supposed to actually do with this information?"

so if we have a concrete example lets list it, if we don't lets take it out to have less text

Reply:
Say you do a key-binding using the arrow keys and you enter it on a macOs platform - that binding will not be usable on any other OS as they do not set the keypad modifier - so you then click on the keypad modifier to remove it - the binding will then work on all platforms.

Or - you are writing a package for a German user and need to set a binding on, say ß  - from my UK (Linux OS) keyboard I can only produce that character by using the `AltGr` as a compose key { `<AltGr>+s`} so that my key-binding will have the `GroupSwitch` modifier - for it to work in a package for a German user with their keyboard having that key to the right of the `0` I will need to remove that modifier...

:thinking: Remember that setting a modifier here means it is a *requirement* for the binding to fire - OTOH we really ought to have these as tri-states (with a *default* to the tristate for *don't care* and then unchecked for *must not be present* and checked for *must be present*)

Anyhow - this sort of tweaking is impossible with the existing code...

Reply:
I agree that the differences in keyboards make it a complex issue, but I think we should solve it for the users instead of moving the problem over to them.

I think I have seen games that have an option to take care of this automatically - I'll see if I can dig it up... they store keybindings as the letters that they mean instead of the actual numerical codes, I believe.

Reply:
Well the existing code is useful because on at least some OSes (- but not, it seems, FreeBSD) it allows the use of many of all those extra (multi-media and other) keys around the edges of the keyboard...